src/ex1/script.js

```diff
@@ -0,0 +1,55 @@
+const input = document.querySelector(".todo-input");
+const addButton = document.querySelector(".todo-button");
+const todoList = document.querySelector(".todo-list");
+
+addButton.addEventListener("click", () => addTodo(input));
+input.addEventListener("keypress", (e) => {
+  if (e.key === "Enter") {
+    addTodo(input);
+  }
+});
+
+function addTodo(input) {
+  if (input.value.length === 0) {
+    changeInputPlaceholder(false);
+    return;
+  }
+  changeInputPlaceholder(true);
+  const todoItem = document.createElement("div");
+  const newTodo = document.createElement("li");
+  const deleteButton = document.createElement("button");
+  const trashIcon = document.createElement("i");
+
+  newTodo.taskName = input.value;
+  newTodo.innerHTML = newTodo.taskName.substring(0, 20);
+  if (newTodo.taskName.length > 20) {
+    newTodo.innerHTML += "...";
+  }
+
+  trashIcon.classList.add("fas", "fa-trash");
+  deleteButton.append(trashIcon);
+  deleteButton.classList.add("delete-btn");
```

Comment:
you have here a lot of createElement, append, and class that you are adding 
in this case I would just add it all together to the innerHtml in one html structure
this why it is more clear what is the hierarchy and what is actually being rendered